hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,20 @@
+RELEASE_TYPE: minor
+
+This release adds a warning for tests that set the global PRNG state, for example
+by using :func:`python:random.seed` without appropriate cleanup (:issue:`1919`).
+While convenient, this creates significant cross-test state and can substantially
+reduce Hypothesis' chances of finding new bugs over multiple runs!
+
+If you want deterministic Hypothesis tests - though we strongly recommend varied
+generation and :doc:`reproducing` instead - you can use the
+:obj:`~hypothesis.settings.derandomize` setting.  For other tests, the following
```

Comment:
Perhaps `derandomize` or the `seed` decorator?

Reply:
Our general advice has been that the `derandomize` setting is a configuration choice (that we don't recommend), and `@seed` is for reproducing failures but not for long-term use; I'm basically just following the docs here.

Reply:
Fair!